app/util/util_scrape.py

```
@@ -12,3 +21,32 @@ def safe_extract_attr(element, selector: str, attr: str) -> str:
+    except (AttributeError, TypeError):
+        return ""
+    except NoSuchElementException as e:
+        logger.warning(f"No such attr in element, {e}")
+        return ""
+
+
+def clean_url(url: str) -> str:
+    try:
+        parsed = urlparse(url)
+        return f"{parsed.scheme}://{parsed.netloc}{parsed.path}"
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)